development/cpp/configuring_an_ide/clion.rst

```diff
@@ -42,3 +42,27 @@ which can also be used by CLion.
 
 If you run into any issues, ask for help in one of
 `Godot's community channels <https://godotengine.org/community>`__.
+
+Debugging the project
+---------------------
+
+Since CLion does not support scons, you won't be able to compile, launch, and debug Godot from CLion in one step.
+You will first need to `compile godot yourself <https://docs.godotengine.org/en/stable/development/compiling/index.html>`__ and run the binary without CLion. You will then be able to debug Godot by using the `Attach to process <https://www.jetbrains.com/help/clion/attaching-to-local-process.html>`__ feature.
+
+- Run the compilation with the debug flag like so : ``scons platform=x11 target=debug`` (replace x11 by your platform)
+
+- Run the binary you have created (in the bin directory). If you want to debug a specific project, run the binary with the following arguments : ``--editor --path path/to/your/godot/project``. To run the project instead of editing it, remove the "--editor" argument.
```

Comment:
```suggestion
- Run the binary you have created (in the bin directory). If you want to debug a specific project, run the binary with the following arguments : ``--editor --path path/to/your/godot/project``. To run the project instead of editing it, remove the ``--editor`` argument.
```